torchbox/torchbox.com
# [5988] Carbon aware badge

[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1426625988)

### Description of Changes Made

- Added a new component for the carbon emissions copy in the footer to match designs (as much as possible)
- This also adds a new grey colour variable for the component's border colour, which didn't exist before. The background and border colours seem to remain static regardless of theme, so I've used the sass variable for these.

**Note**: I have added an underline to the link(s) which isn't in the designs - this is just to visually differentiate between the bold, non-linked text and the linked text which will eventually go to the content page, otherwise it wasn't clear that the carbon measurement text was a link.
**Also note**: I've added some fallback text for the JS-generated page speed. This will need reviewing - but it might not be needed if we decide to remove the page speed anyway.


Currently this is all hard coded until we have 
- content fields in the CMS
- content page to link to

### How to Test

- Open any page locally and view the footer (or check the [pattern library component](http://localhost:8000/pattern-library/pattern/patterns/molecules/carbon-impact/carbon-impact.html))

### Screenshots

<details>
  <summary>Expand to see more</summary>

![tbx-page-carbon](https://github.com/torchbox/torchbox.com/assets/8680557/465c1069-4a12-418e-b1cb-16f81b92a27d)

</details>

### MR Checklist


#### Unit tests


#### Documentation


#### Browser testing

  - Latest version of Chrome on mac
  - Latest version of Firefox on mac
  - Latest version of Safari on mac

#### Data protection


#### Accessibility


#### Sustainability

- [x] If JavaScript is needed alternatives have been considered - **may need further refinement depending on if we use the JS or not**

#### Pattern library
